src/main/charts/bamboo-agent/Changelog.md

```diff
@@ -14,7 +14,7 @@
 
 ## 1.3.0
 
-**Release date:** 2022-23-03
+**Release date:** 2022-03-03
```

Comment:
```suggestion
**Release date:** 2022-03-24
```

Reply:
The release happened in March 23 (more precise is March 24 based on GitHub release history)

Reply:
Please check for other products as well.